README.md

```diff
@@ -2,9 +2,13 @@
 
 [![Join the chat at https://gitter.im/carlosdarienvizcaino/SwingSoccer](https://badges.gitter.im/carlosdarienvizcaino/SwingSoccer.svg)](https://gitter.im/carlosdarienvizcaino/SwingSoccer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
We can remove this for now. This gitter chat is not actually used and if we do decide on using one in future it can be a new chat room